portfolio/src/main/java/com/google/sps/model/Comment.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.model;
+
+import java.util.Date;
+
+public class Comment {
+  private long id;
+  private String content;
+  private String userName;
+  private Date time;
+  private long like;
+
+  public Comment(long id, String userName, String content, Date time, long like) {
```

Comment:
it would be nice to have a static factory method that creates a comment from the DB entry, this way you could make this constructor private. A factory method like this make the construction safer and easier to update in the future.

Reply:
Good suggestion! Done